calicoctl/commands/node/diags.go

```diff
@@ -222,6 +223,13 @@ func writeDiags(cmds diagCmd, dir string) {
 	content, err := exec.Command(parts[0], parts[1:]...).CombinedOutput()
 	if err != nil {
 		fmt.Printf("Failed to run command: %s\nError: %s\n", cmds.cmd, string(content))
+		if strings.Compare(cmds.cmd, "iptables-save -c") == 0 {
```

Comment:
If you want to do exact equality, I think it's more common in go to just do `==`. 

e.g., `cmds.cmd == "iptables-save -c"`